gadget/ondisk.go

```diff
@@ -131,15 +131,10 @@ type OnDiskStructure struct {
 // OnDiskVolume holds information about the disk device including its partitioning
 // schema, the partition table, and the structure layout it contains.
 type OnDiskVolume struct {
+	LaidOutVolume
```

Comment:
now we have double structures, not sure it's a win. This should probably just drop the the unexported field for now.

Reply:
This was intended to be an intermediate step to be followed by replacing OnDiskVolume with LaidOutVolume with the extra needed fields added, but we can go back and only drop the partition table as the first step.

Reply:
Done.